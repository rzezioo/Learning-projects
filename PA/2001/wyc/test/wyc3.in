1010 100
-1 -39
-1 82
-1 85
-1 -96
-1 -24
-1 55
-1 -16
-1 -41
-1 48
-1 -84
-1 -7
1 -93
1 32
1 -95
1 -36
1 -13
1 -30
1 100
1 63
-2 98
-2 45
-2 -56
-2 -69
-2 74
-2 82
-2 -88
-2 77
-2 39
2 43
2 41
2 14
2 63
2 -90
2 54
-3 49
-3 -59
-3 -78
-3 91
-3 85
-3 39
-3 97
3 -85
3 77
3 -62
3 -30
3 -73
3 -90
3 -92
3 4
-4 11
-4 6
-4 -90
-4 -72
-4 -87
4 8
4 -66
4 -70
4 -69
4 50
4 -77
-5 48
-5 -51
-5 70
-5 44
-5 23
-5 -67
-5 -74
5 -62
5 -13
-6 18
6 -79
6 -70
6 75
6 53
6 -65
6 92
6 -28
6 -99
6 -9
6 -14
6 -48
6 -68
6 -34
6 72
6 -80
6 90
6 95
6 -10
6 -7
6 30
6 39
6 -11
6 -85
-7 32
-7 -93
-7 -58
-7 15
-7 11
7 55
7 -98
7 97
7 19
7 -79
7 69
7 -99
7 23
7 -32
7 47
7 -84
7 -81
7 -20
-8 50
-8 -56
-8 -53
-8 -24
-8 76
-8 -59
-8 51
-8 45
-8 39
-8 49
-8 -48
8 41
8 9
8 -95
8 -39
-9 32
-9 -42
-9 29
-9 14
-9 -13
-9 -95
-9 -30
-9 43
9 -46
9 61
9 -64
9 53
9 -34
9 -47
9 -79
9 92
9 -37
9 30
9 56
9 72
9 60
9 95
9 -68
9 -35
9 -82
9 -10
-10 -22
-10 -62
-10 14
-10 18
10 66
10 90
10 -47
10 -15
10 38
10 60
10 -81
10 -17
10 -80
10 -64
10 -65
10 75
10 -49
10 44
-11 63
11 -94
11 58
11 -41
11 -34
11 39
11 56
11 -100
11 -56
11 16
11 26
11 60
11 55
11 -81
11 -67
-12 -39
-12 50
-12 77
-12 -86
-12 67
-12 21
-12 -28
12 29
12 -95
12 54
-13 -34
-13 61
-13 -80
-13 55
-13 -40
-13 -35
-13 16
-13 -94
-13 93
-13 -17
-13 38
-13 -49
-13 -68
-13 95
13 -95
13 32
13 63
-14 -92
14 -50
14 97
14 -100
14 -64
14 86
14 -79
14 53
14 66
14 -77
14 26
14 -54
14 30
14 19
14 -35
14 90
14 55
-15 29
-15 -36
-15 -95
-15 -92
15 83
15 55
15 -89
15 -100
15 75
15 -79
15 72
15 26
15 -29
15 -35
15 60
15 42
15 -65
15 52
15 -54
15 53
15 -57
15 71
-16 28
-16 33
-16 88
-16 53
-16 96
-16 59
-16 75
-16 24
16 43
16 -42
-17 41
-17 29
-17 96
-17 -92
-17 -42
-17 -22
-17 -38
17 60
17 31
17 66
17 69
17 -94
17 -40
17 71
17 52
17 53
17 -33
17 -57
17 -64
17 -35
17 75
17 -50
-18 54
-18 96
-18 63
-18 -95
18 -40
18 -94
18 -81
18 87
18 26
18 -50
18 85
18 52
18 -57
18 55
18 -32
18 95
18 -68
18 -65
18 92
-19 45
-19 50
-19 -44
-19 -24
-19 -36
-19 51
-19 26
-19 59
-19 -48
19 -42
-20 -53
-20 -39
-20 59
-20 28
-20 -48
-20 -40
20 -38
20 43
20 32
20 29
20 41
20 96
-21 -56
-21 -78
-21 -26
-21 82
-21 -44
-21 -97
-21 -55
-21 51
-21 91
-21 45
-21 -59
-21 -28
21 -58
21 -30
21 -72
21 -97
-22 -43
-22 -94
-22 31
-22 -64
-22 39
-22 37
-22 86
-22 -68
-22 55
-22 97
22 -58
22 100
-23 -25
-23 44
-23 -67
-23 -91
-23 71
23 100
23 -30
23 43
23 54
23 29
-24 70
-24 75
24 -92
24 -87
-25 -75
-25 -31
-25 28
-25 -70
-25 -78
-25 45
-25 -46
25 -38
25 -62
25 -92
25 -93
-26 89
-26 51
-26 -85
-26 -66
-26 76
-26 -48
26 29
26 41
26 96
26 -90
-27 -39
-27 -68
-27 47
-27 84
-27 -88
-27 59
-27 28
-27 89
-27 -44
-27 -48
-27 70
27 -30
27 29
-28 -97
-28 -95
-28 29
-28 -92
-28 -58
-28 -73
-28 -90
28 74
28 50
28 -66
28 -55
28 -69
28 -86
28 98
28 -44
28 -48
28 88
-29 -58
-29 -62
29 -68
29 71
29 -46
29 -34
29 83
29 95
29 -41
29 -99
29 -43
-30 -67
-30 -96
-30 66
-30 -57
-30 -41
-30 -79
-30 72
-30 -99
-30 58
-30 39
-30 60
-30 36
-30 95
-30 -45
-30 92
30 63
30 100
30 -61
-31 -51
-31 48
-31 88
-31 82
-31 -76
31 -95
31 54
31 -87
31 -73
31 80
31 -53
-32 -93
-32 96
-32 -95
32 -35
32 -98
32 -33
32 -46
32 -41
32 -96
32 55
32 -94
32 75
32 -99
32 -68
-33 -97
-33 80
-33 70
33 82
33 81
33 47
33 -36
33 76
33 -56
33 -44
33 51
33 -88
33 70
-34 35
-34 80
-34 -61
-34 46
-34 -42
-34 54
-34 40
34 44
34 -40
34 81
34 -70
34 -46
34 -83
34 89
34 -67
-35 -92
-35 -87
-35 -95
35 -79
35 -49
35 -74
35 -82
35 -94
35 -57
35 -64
35 48
35 -48
35 60
35 -40
35 55
-36 75
-36 58
-36 -40
-36 97
-36 -49
-36 -50
-36 71
36 -90
-37 44
-37 -72
-37 -45
37 98
37 -83
37 -86
37 64
37 68
37 -71
37 45
37 91
37 -78
-38 66
-38 56
-38 -98
-38 -46
-38 -44
-38 -57
-38 -64
-38 -68
-38 -40
38 43
38 -93
38 41
-39 76
-39 49
-39 67
-39 -71
-39 74
-39 -44
39 41
39 -72
-40 -62
-40 46
40 88
40 44
40 81
40 -70
-41 -90
-41 54
-41 -61
41 -43
41 -79
41 -64
41 -80
-42 -47
-42 78
-42 51
-42 60
-42 -57
-42 66
42 43
42 -95
42 -93
42 -62
42 100
-43 -58
-43 -62
-43 -73
43 -65
43 -63
43 -98
43 -99
43 73
43 -64
43 87
43 -46
43 75
43 -94
43 -77
43 66
43 62
-44 -72
-44 -87
-44 59
-44 88
-44 -51
44 -56
44 45
44 84
44 67
44 51
44 -66
44 74
44 -75
44 -71
44 76
-45 -83
-45 -66
-45 -92
-45 -52
-45 91
-45 -71
45 -93
45 59
45 -49
45 -98
-46 -90
-46 -61
-46 -87
-46 -92
-46 -58
-46 -95
46 -70
46 -71
-47 80
-47 -95
-47 63
-47 -75
47 77
47 75
47 88
47 -51
47 48
47 53
47 66
-48 79
-48 -97
-48 64
-48 70
-48 57
48 100
48 67
48 51
48 -66
48 -71
48 -70
-49 -60
-49 57
-49 98
-49 -69
-49 -71
-49 84
-49 -90
-49 81
-49 -53
-49 82
49 -64
49 96
49 88
49 -51
49 59
49 77
49 69
49 -82
49 -65
49 -62
-50 64
-50 -71
-50 98
-50 -83
-50 94
-50 74
-50 84
-50 89
-50 81
-50 -56
50 59
50 78
50 -74
50 56
50 -81
50 -76
-51 -66
-51 89
-51 -86
-51 -56
-51 82
-51 98
-51 -78
51 77
51 88
51 -72
-52 -76
-52 55
-52 83
-52 -94
-52 71
52 -61
52 100
52 -73
52 -60
52 -92
52 54
52 -87
52 96
52 -95
52 68
-53 70
-53 -84
-53 -82
-53 75
-53 -81
53 100
53 -95
53 -73
53 -61
53 -58
53 99
-54 -92
-54 96
54 93
54 55
54 87
54 73
54 62
-55 -94
-55 78
-55 -65
-55 -68
-55 -98
55 -61
55 100
55 96
55 63
55 -95
55 64
55 80
55 -90
55 94
55 -73
55 -60
55 65
-56 78
-56 -82
-56 -74
56 94
56 89
56 -83
56 -71
56 80
56 74
-57 64
-57 63
-57 -61
-57 -93
57 83
57 -89
57 -79
57 -94
57 66
57 70
57 -64
57 -91
-58 -94
-58 95
-58 73
-58 -64
-58 -79
-58 92
-58 62
-58 -74
-58 -82
58 -73
58 100
-59 -85
-59 -72
59 84
59 76
59 67
59 -83
59 -75
59 98
59 -86
-60 78
-60 -79
-60 -89
-60 -94
-60 86
-60 -91
-60 -98
-60 -68
-60 -83
-60 -81
60 -93
60 -73
60 100
60 -95
60 99
60 79
60 -90
60 96
60 80
-61 72
-61 -80
-61 66
-61 -84
-61 -68
61 63
-62 75
-62 93
-62 91
-62 95
-62 -99
-62 -64
-62 -82
62 -93
62 -87
62 63
62 100
-63 -95
63 73
63 -68
63 87
63 93
63 95
63 -65
63 75
63 -98
-64 -95
-64 -73
-64 -90
-64 -92
-64 -87
-64 100
64 69
64 83
64 -94
64 -79
64 -68
64 -98
64 -81
64 88
-65 80
-65 99
-65 -90
-65 -93
-65 79
-65 68
65 69
65 -68
65 -94
65 -67
65 -66
65 71
-66 78
-66 86
-66 -84
-66 -72
-66 -97
-66 83
-66 -85
66 79
66 68
66 99
66 94
66 89
-67 -78
-67 81
-67 -83
-67 68
-67 89
-67 82
-67 91
67 -76
67 78
-68 79
-68 99
-68 94
68 71
-69 -84
-69 78
-69 -91
-69 86
-69 71
-69 83
-69 -98
-69 -89
69 89
69 -83
69 98
69 -86
69 99
69 -71
-70 -97
70 -71
-71 78
-71 83
-71 -84
-71 -74
71 -93
71 -86
71 -90
71 91
71 -78
-72 -83
-72 -86
-72 85
-72 81
-72 -77
72 77
72 -85
-73 -78
-73 92
-73 -77
-73 -99
73 80
73 -90
73 -87
-74 81
-74 -86
74 -81
74 -76
74 -85
74 88
74 -82
-75 88
75 -92
75 -93
75 96
75 -95
-76 81
-76 -78
-76 91
-76 82
-76 84
-76 -86
-76 99
-76 -87
-76 98
76 88
76 -82
-77 -90
-77 -85
77 89
77 -78
77 82
-78 -97
-78 -84
-78 -82
-78 -81
78 -86
78 89
78 -83
-79 -90
-79 -87
-79 -92
-79 100
-79 -95
-80 -87
80 96
-81 89
-81 -86
-81 91
81 -85
81 88
81 -82
82 -95
82 88
82 -85
-83 -98
-83 -91
-83 -84
-83 -85
-83 86
83 -87
-84 -86
-84 98
-84 99
-85 91
-86 -91
86 98
86 -87
86 100
-87 -88
-87 -98
87 100
87 96
87 -93
89 -91
89 -98
-90 95
-90 93
90 -93
90 -92
-91 -93
-91 99
-91 -95
-91 98
91 -97
-92 95
92 -95
-93 -94
-94 -95
-94 99
-95 -99
-95 -100
96 -99
