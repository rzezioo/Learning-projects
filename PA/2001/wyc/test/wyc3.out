50
4 6 7 9 10 11 14 15 17 18 28 29 32 33 34 35 40 41 43 45 46 47 48 49 50 51 54 57 59 63 64 65 67 68 70 74 76 77 79 80 81 82 84 89 91 94 96 98 99 100
